graphql/task_resolver.go

```diff
@@ -544,7 +544,7 @@ func (r *taskResolver) ProjectIdentifier(ctx context.Context, obj *restModel.API
 func (r *taskResolver) SpawnHostLink(ctx context.Context, obj *restModel.APITask) (*string, error) {
 	host, err := host.FindOne(ctx, host.ById(*obj.HostId))
 	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error finding host for task %s", *obj.Id))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding host for task %s", *obj.Id))
```

Comment:
`utility.FromStringPtr`
```suggestion
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding host for task '%s'", *obj.Id))
```